README.md

```diff
@@ -29,25 +29,28 @@ Given the following `flake.nix` example:
   description = "An example project.";
 
   inputs.pre-commit-hooks.url = "github:cachix/pre-commit-hooks.nix";
-  inputs.flake-utils.url = "github:numtide/flake-utils";
-
-  outputs = { self, nixpkgs, pre-commit-hooks, flake-utils }:
-    flake-utils.lib.eachDefaultSystem (system:
-      {
-        checks = {
-          pre-commit-check = pre-commit-hooks.lib.${system}.run {
-            src = ./.;
-            hooks = {
-              nixpkgs-fmt.enable = true;
-            };
-          };
-        };
-        devShell = nixpkgs.legacyPackages.${system}.mkShell {
-          inherit (self.checks.${system}.pre-commit-check) shellHook;
-          buildInputs = self.checks.${system}.pre-commit-check.enabledPackages;
+
+  outputs = { self, nixpkgs, ... }@inputs:
+    let
+      # NOTE: you *can* use `nixpkgs.lib.genAttrs [ … ]` to loop thru
+      # multiple systems (where `nixpkgs.lib.systems.flakeExposed` is
+      # the default system list for flakes).
+      system = "x86_64-linux";
+    in
+    {
+    checks = {
+      pre-commit-check = inputs.pre-commit-hooks.lib.${system}.run {
+        src = ./.;
+        hooks = {
+          nixpkgs-fmt.enable = true;
         };
-      }
-    );
+      };
+    };
+    devShell = nixpkgs.legacyPackages.${system}.mkShell {
+      inherit (self.checks.${system}.pre-commit-check) shellHook;
+      buildInputs = self.checks.${system}.pre-commit-check.enabledPackages;
+    };
+  }
```

Comment:
Lets revert this.

1. This isn't a valid flake.
2. People copy-paste these snippets all the time. It should at least evaluate on all 4 default systems out of the box. We get enough support requests as it is!
3. We're not the place to teach people how to use flakes. The main point here is to show a reasonable flake config to get people started with git hooks.

Reply:
> My concern is folks copying the snippet then all of their downstream users will have a probably-unnecessary dependency to audit from copying code from the repository.

A laudable cause, but it feels as if we're trying to patch the design deficiencies of flakes by making everyone else miserable.

> Are you sure this is preferred to setting the example without `flake-utils` with a `nixpkgs.lib.genAttrs nixpkgs.lib.systems.flakeExposed`? 

Honestly, I don't think it really matters in the grand scheme of things. Again, this is an example meant to showcase the pre-commit/git-hooks bits.

We could do a `forAllSystems = nixpkgs.lib.genAttrs systems`, where `systems = [ "x86_64-linux" "x86_64-darwin" "aarch64-linux" "aarch64-darwin" ];`? This is generally what we do across cachix repos. `flakeExposed` seems like way too much, no?
```
nix-repl> lib.systems.flakeExposed
[ "x86_64-linux" "aarch64-linux" "x86_64-darwin" "armv6l-linux" "armv7l-linux" "i686-linux" "mipsel-linux" "aarch64-darwin" "armv5tel-linux" "powerpc64le-linux" "riscv64-linux" ]
```